packages/android_intent_plus/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,17 @@
     -->
   <uses-permission android:name="android.permission.INTERNET" />
 
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="http" />
+        </intent>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https" />
+        </intent>
+    </queries>
+
```

Comment:
I guess the intention is that the example app appears listed when calling to resolve activity, right?

Reply:
The example app uses getResolvedActivity to query activities that can handle http urls. In order to query those activities these query entries are required

Reply:
Ah thanks, I see https://developer.android.com/guide/topics/manifest/queries-element

This should be added to the README as well.

Reply:
There is already a section in the readme called `Android 11 package visibility`